Dockerfile

```diff
@@ -1,8 +1,17 @@
-FROM mhart/alpine-node:10.5
+FROM mhart/alpine-node:14.4
 MAINTAINER Paul Bill <paul.bill@bravissimo.com>
 
 RUN apk add --no-cache bash
```

Comment:
We should probably merge the two `RUN` commands to reduce the number of layers to the image

Reply:
Done 👍